tests/components/media_player/test_async_helpers.py

```diff
@@ -1,6 +1,6 @@
 """The tests for the Async Media player helper functions."""
```

Comment:
The filename and title are misleading as the tests are not only for the `Async Media player helper functions` but for Async and Sync. I get that this is a straight conversion from unittest to pytest so far. But fixing this should be part of this pr.

Reply:
Thanks for reviewing. I have split the file into two (one for async, one  for sync). This will reduce the line number of each file and removes the confusion with the filename/title and content.

Reply:
We should really just move all these tests to `test_init.py` module. But that's not required in this PR.